tests/session/cmd/test_legacy.py

```diff
@@ -28,6 +28,21 @@ def test_legacy_list_default(tox_project: ToxProjectCreator, mocker: MockerFixtu
     assert outcome.state.options.show_core is False
 
 
+@pytest.mark.parametrize(
+    "configuration",
+    [
+        pytest.param("", id="missing envlist"),
+        pytest.param("[toxenv]", id="empty envlist"),
```

Comment:
Both of these are missing. An empty env list would look like this:
`[toxenv]\nenv_list=`